data/categories.txt

```diff
@@ -115,11 +115,12 @@ ru:4Продукты|Еда
 ar:طعام|بقالية
 be:4Прадукты|Ежа
 bg:4Продукти|Храна
+ca:Queviures|Provisions
```

Comment:
Thanks! Did you check that here and below the first keyword matches related translations in strings.txt (in this case, `[food]`)?

Reply:
I've added another commit.